web/styles/popovers.css

```diff
@@ -895,14 +928,18 @@ ul {
 #giphy_grid_in_popover {
     /* 300px of GIPHY grid + 5px is the extra gutter space
      * between gif columns. */
-    width: 305px;
+    width: 307px;
```

Comment:
The comment just above indicates that the width should be `305px`. Please fix the mismatch.